src/shared/parameter/config_definitions/ssl_simulation_config.yaml

```diff
@@ -0,0 +1,27 @@
+- int:
+    name: team_channel
+    min: 0
+    max: 15
+    value: 0
+    description: >-
+      The multicast channel to use to communicate with the blue robots
```

Comment:
is this meant to be for friendly robots, regardless whether they are blue or yellow?

Reply:
Oops supposed to be friendly robots, fixed